eval/na/zoned/eval_framework.py

```
@@ -0,0 +1,661 @@
+        The compiled code and stats.
+    """
+    return compiler.compile(qc), compiler.stats()
+
+
+def process_benchmark(
+    compiler: RoutingAwareCompiler | RoutingAwareNativeGateCompiler,
+    setting_name: str,
+    qc: QuantumComputation,
+    benchmark_name: str,
+    evaluator: Evaluator,
+    drop_u_gates: bool = False,
```

Ruff (0.15.20)

[warning] Boolean-typed positional argument in function definition

(FBT001)